src/include/daos_srv/security.h

```diff
@@ -88,21 +88,21 @@ ds_sec_pool_get_capabilities(uint64_t flags, d_iov_t *cred,
  * \param[in]	ownership	Container ownership information
  * \param[in]	acl		Container ACL
  * \param[out]	capas		Capability bits for this user
+ * \param[out]	is_owner	If this user is the owner
  *
  * \return	0		Success
  *		-DER_INVAL	Invalid input
  *		-DER_NOMEM	Out of memory
  */
 int
-ds_sec_cont_get_capabilities(uint64_t flags, d_iov_t *cred,
-			     struct d_ownership *ownership,
-			     struct daos_acl *acl, uint64_t *capas);
+ds_sec_cont_get_capabilities(uint64_t flags, d_iov_t *cred, struct d_ownership *ownership,
+			     struct daos_acl *acl, uint64_t *capas, bool *is_owner);
```

Comment:
I'm a little reluctant to return is_owner from the capability checks, it feels a bit like overloading to me. I don't have anything against exposing the acl_user_is_owner function.